runtime/gc_vlhgc/ProjectedSurvivalCollectionSetDelegate.cpp

```diff
@@ -540,7 +544,7 @@ MM_ProjectedSurvivalCollectionSetDelegate::rateOfReturnCalculationBeforeSweep(MM
 				if(!region->getRememberedSetCardList()->isAccurate()) {
 					stats->_reclaimStats._regionCountOverflow += 1;
 				}
-			} else if(region->isArrayletLeaf()) {
+			} else if(region->isArrayletLeaf() && !_extensions->isVirtualLargeObjectHeapEnabled) {
 				MM_HeapRegionDescriptorVLHGC *parentRegion = (MM_HeapRegionDescriptorVLHGC *)_regionManager->regionDescriptorForAddress((void *)region->_allocateData.getSpine());
```

Comment:
put a comment that recycling of regions reserved for offheap objects is already done in root clearable phase at the end of GMP

Reply:
sorry, not here, but in Sweep code